geometry_msgs/Pose location
long_term_deployment/Task task
time arrival_time
time departure_time
time duration
